dvc/project.py

```diff
@@ -165,10 +165,22 @@ def install(self):
     def to_dvc_path(self, path):
         return os.path.relpath(path, self.root_dir)
 
-    def _check_output_duplication(self, outs):
+    def _check_cwd_specified_as_output(self, cwd, stages=None):
+        from dvc.exceptions import WorkingDirectoryAsOutputError
+
+        stages = stages or self.stages()
+
+        for stage in stages:
+            for output in stage.outs:
+                if os.path.isdir(output.path) and output.rel_path == cwd:
```

Comment:
cwd is not guaranteed to be in relative path notation, it can be in absolute notation as well. We need to normalize paths before comparing them.